.github/workflows/botonic-core.tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Botonic core tests
+ 
+on: [push]
+ 
+jobs:
+  botonic_react_tests:
+    name: Botonic core tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checking out to current branch (Step 1 of 6)
+      uses: actions/checkout@v1
+    - name: Setting up node (Step 2 of 6)
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12'
+    - name: Setting up cache (Step 3 of 6)
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Install botonic-core (Step 4 of 6)
+      run: (cd ./packages/botonic-core && npm install)
```

Comment:
Initially I thought that we should verify that we can build from package.json creating the lock.json file with "npm i -D"
,but I have mixed feelings. Let's keep it simple and leave it as it is :+1: